cmd/genji/main.go

```diff
@@ -26,38 +25,91 @@ func main() {
 		},
 	}
 
+	app.Commands = []*cli.Command{
+		{
+			Name:      "insert",
+			Usage:     "Allows inserting documents from the command line",
+			UsageText: "genji insert [options] [arguments...]",
+			HideHelp:  true,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "engine",
+					Aliases:  []string{"e"},
+					Usage:    "Store engine bolt or badger",
+				},
+				&cli.StringFlag{
+					Name:     "db",
+					Usage:    "path of the database file",
+					Required: true,
+				},
+				&cli.StringFlag{
+					Name:     "table",
+					Aliases:  []string{"t"},
+					Usage:    "name of the table, it must already exist",
+					Required: true,
+				},
+			},
+			Action: func(c *cli.Context) error {
+				dbPath := fmt.Sprintf("%v", c.Value("db"))
+				table := fmt.Sprintf("%v", c.Value("table"))
+				// Use bolt as default engine.
+				engine := "bolt"
+				if c.IsSet("engine") {
+					engine = fmt.Sprintf("%v", c.Value("engine"))
+				}
+
+				data := c.Args().Slice()
+				err := shell.RunInsertCommand(engine, dbPath, table, data)
+				// Print error but return nil to avoid double error printing.
```

Comment:
Why not just returning the error and let the framework print it?

Reply:
If we return the error, it will be print by `log.Fatal(err)` and the datetime is printed. I find it weird. So what do you think? Should I return `err` and use `log.Fatal(err)` ?

Reply:
I understand, in that case how about replac
ing `log.Fatal` by `fmt.Fprintln(os.Stderr, err)`

Reply:
It what I did, I replaced `log.Fatal` by `fmt.Println(err)` but I don't told you why. Should I change it, by `fmt.Fprintln(os.stderr, err)`?